themes/default/content/docs/guides/saml/gsuite.md

```diff
@@ -76,14 +76,14 @@ domain users by selecting the down arrow in the **User access** panel:
     ![Enable the SAML application part 2](/images/docs/reference/service/saml-gsuite/gsuite-app-enable-2.png)
 
    At this point, you're done configuring Google Workspace, and can move on to completing SAML SSO setup in
-   the Pulumi Service.
+   the Pulumi Cloud.
 
 ## Configuring Your Pulumi Organization
 
 The final step in the process consists of associating your Pulumi organization with your SSO identity
 provider.
 
-1. Sign in to the Pulumi Service where your SAML organization resides, then navigate to the **Settings** tab for that
+1. Sign in to the Pulumi Cloud where your SAML organization resides, then navigate to the **Settings** tab for that
```

Comment:
```suggestion
1. Sign in to the Pulumi Cloud console where your SAML organization resides, then navigate to the **Settings** tab for that
```

Reply:
@jkodroff lets leave this as it was before this pr opened, we can come back later if we establish a standard around using console.